pkg/solana/types.go

```diff
@@ -162,21 +162,12 @@ type TransmissionV1 struct {
 // CL Core OCR2 job spec RelayConfig member for Solana
 type RelayConfig struct {
 	// network data
-	NodeEndpointHTTP string `json:"nodeEndpointHTTP"`
+	ChainID  string `json:"chainID"`  // required
+	NodeName string `json:"nodeName"` // optional, defaults to a random node with ChainID
```

Comment:
Just the ability to specific a single node by name. That is not available on EVM, and it also conflicts with the new EVM failover, which monitors the health of the nodes in a pool and swaps them in and out behind a single client. We could merge the two in some way of course (e.g. ordered csv of names plus fallback to random), and we may have to eventually :shrug:

Reply:
Auto-failovers are something that we'll need to do in the short term, and then the question is what's the use case of pointing a specific job to go through a specific RPC node, and how will that work in the context of auto-failovers 🤔

I vote to simplify, remove this option for now, and let's revisit in the context of auto-failovers.

Reply:
👍🏼 we can remove it from the spec

so my understanding is that it will basically pick a random node/endpoint each time?
or would this change propagate further and only allow one node to be configured per chain?

Reply:
removed node name: https://github.com/smartcontractkit/chainlink-solana/pull/250/commits/8d722896e06dd07f7be80fd5bb4ade83843ba986